README.md

```diff
@@ -1,2 +1,40 @@
-# identity-authentication-flow-composer
-Authentication Flow Composer is a UI based tool to design and configure the authentication flow easily.
+# Authentication Flow Composer
+Authentication Flow Composer is a UI-based tool to design and configure the authentication flows for the WSO2 Identity
+Server. This approach allows users to define the authentication flows without any prior syntax knowledge.
+
+### Setup Identity Server for the application
+
+* Add the following code to `<IS_HOME>/repository/conf/deployment.toml` in WSO2 Identity Server to allow CORS.
+  In this guide, the installation location of WSO2 Identity Server is referred to as `<IS_HOME>`.
+```toml
+    [cors]
+    allowed_origins = [
+       "http://localhost:3000"
+    ]
+    supported_methods = [
+       "GET",
+       "POST",
+       "HEAD",
+       "OPTIONS",
+       "PUT",
+       "PATCH",
+       "HEAD",
+       "DELETE",
+       "PATCH"
+    ]
+    exposed_headers = [ "Location" ]
+```    
+
+    
+* Start the Identity Server using following commands from `<IS_HOME>/bin`.
```

Comment:
Lets link to official docs for instructions.

https://is.docs.wso2.com/en/latest/setup/running-the-product/#starting-on-windowslinuxmac-os

Reply:
Fixed with [5e702ed](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/11/commits/5e702ed5f05afdecdf92862e264eb12548626f48).